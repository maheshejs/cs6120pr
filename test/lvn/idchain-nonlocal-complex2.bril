@main {
  x: int = const 4;
  copy0: int = id x;
  jmp .label;
.label:
  copy1: int = id copy0;
  copy0: int = const 7;
  copy2: int = id copy1;
  copy3: int = id copy2;
  print copy3;
}
