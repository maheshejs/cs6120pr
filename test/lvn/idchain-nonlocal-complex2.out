@main {
  x: int = const 4;
  copy0: int = const 4;
  jmp .label;
.label:
  copy1: int = id x;
  copy0: int = const 7;
  copy2: int = id x;
  copy3: int = id x;
  print x;
}
